src/style-spec/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.28.0-beta.1",
+  "version": "13.28.0",
```

Comment:
I don't see an entry for this version in [style-spec/CHANGELOG](https://github.com/mapbox/mapbox-gl-js/blob/adf88f9fdfdcc51d3b237efc927b14ff061498ec/src/style-spec/CHANGELOG.md), did we miss this in the beta?

Reply:
No, there were no change relevant to put in the changelog.